bap-builder/PackageMode.go

```diff
@@ -224,61 +264,84 @@ func buildAndCopyPackage(cmdLine *BuildPackageCmdLineArgs, build *[]bringauto_bu
 	if err != nil {
 		return err
 	}
+	logger := bringauto_log.GetLogger()
 
 	for _, buildConfig := range *build {
-		platformString, err := determinePlatformString(&buildConfig)
-		if err != nil {
-			return err
-		}
+		logger.Info("Build %s", buildConfig.Package.GetFullPackageName())
 
 		sysroot := bringauto_sysroot.Sysroot{
 			IsDebug:        buildConfig.Package.IsDebug,
 			PlatformString: platformString,
 		}
 		err = bringauto_prerequisites.Initialize(&sysroot)
-
 		buildConfig.SetSysroot(&sysroot)
+
+		logger.InfoIndent("Run build inside container")
 		err = buildConfig.RunBuild()
 		if err != nil {
 			return err
 		}
 
+		logger.InfoIndent("Copying to Git repository")
+
+		removeHandler := bringauto_process.SignalHandlerAddHandler(buildConfig.CleanUp)
```

Comment:
Why there is no deffer used? What if function returns before the handler is removed.

Reply:
Done.